color.go

```diff
@@ -10,6 +10,8 @@ var (
 	Cyan   = "\033[36m"
 	Gray   = "\033[37m"
 	White  = "\033[97m"
+	Bold   = "\033[1m"
+	Orange = "\033[93m"
```

Comment:
Unfortunately, I can only accept the `Bold` here, because `93` is in fact high intensity yellow, not orange.

Could you also please move the `Bold` line after the `Reset`?